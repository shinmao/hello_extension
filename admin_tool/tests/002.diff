001+ Fatal error: Uncaught Error: Call to undefined function admin_tool_test1() in /var/www/html/so/admin_tool/tests/002.php:2
002+ Stack trace:
001- The extension admin_tool is loaded and working!
002- NULL
003+ #0 {main}
004+   thrown in /var/www/html/so/admin_tool/tests/002.php on line 2