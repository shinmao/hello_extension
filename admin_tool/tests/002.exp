The extension admin_tool is loaded and working!
NULL