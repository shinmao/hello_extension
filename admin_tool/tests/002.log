
---- EXPECTED OUTPUT
The extension admin_tool is loaded and working!
NULL
---- ACTUAL OUTPUT
Fatal error: Uncaught Error: Call to undefined function admin_tool_test1() in /var/www/html/so/admin_tool/tests/002.php:2
Stack trace:
#0 {main}
  thrown in /var/www/html/so/admin_tool/tests/002.php on line 2
---- FAILED
