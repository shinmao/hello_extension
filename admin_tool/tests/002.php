<?php
$ret = admin_tool_test1();

var_dump($ret);
?>
