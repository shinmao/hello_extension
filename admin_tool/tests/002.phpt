--TEST--
admin_tool_test1() Basic test
--SKIPIF--
<?php
if (!extension_loaded('admin_tool')) {
	echo 'skip';
}
?>
--FILE--
<?php
$ret = admin_tool_test1();

var_dump($ret);
?>
--EXPECT--
The extension admin_tool is loaded and working!
NULL
