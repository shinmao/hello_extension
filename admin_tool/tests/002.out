Fatal error: Uncaught Error: Call to undefined function admin_tool_test1() in /var/www/html/so/admin_tool/tests/002.php:2
Stack trace:
#0 {main}
  thrown in /var/www/html/so/admin_tool/tests/002.php on line 2